tests/integration/exec.bats

```diff
@@ -112,3 +112,19 @@ function teardown() {
 
   [[ "${output}" == "uid=1000 gid=1000"* ]]
 }
+
+@test "runc exec --additional-gids" {
+  # --user can't work in rootless containers that don't have idmap.
+  [[ "$ROOTLESS" -ne 0 ]] && requires rootless_idmap
```

Comment:
`--additional-gids` cannot ever work in a rootless container, because `setgroups` is disabled. This should just be `requires root`. In this case, the `specconv` error is correct here.